mods/tsumeta/items.js

```diff
@@ -808,6 +809,24 @@ exports.BattleItems = {
 			}
 			return true;
 		},
+		onStart: function (pokemon) {
+			if (pokemon.baseTemplate.baseSpecies === 'Arceus' || pokemon.baseTemplate.baseSpecies === 'Dunsparce') {
+				let stats = [];
+				for (let stat in pokemon.boosts) {
```

Comment:
Why are we looping through existing boosts to get stats?
Why not just pick from ``['atk', 'def', 'spa', 'spd', 'spe']``

Reply:
iirc that just filters through those stats, can I get some clarification on what you are requesting

Reply:
If your typing to boost a random stat, and you know what the possible stats are, why are we filtering though them to find them? Its unnessecary, not to mention this wont find stats that arent boosted (i think, could be wrong, still unnessecary).

Reply:
I suppose we only need to filter evasion and accuracy out

Reply:
@DeathlyPlays ``if (stat === 'accuracy' || stat === 'evasion') continue;``